src/sidebar/components/test/top-bar-test.js

```diff
@@ -1,15 +1,20 @@
 'use strict';
 
 var angular = require('angular');
+var proxyquire = require('proxyquire');
 
 var util = require('../../directive/test/util');
 
 describe('topBar', function () {
   var fakeSettings = {};
+  var fakeIsThirdPartyService = sinon.stub();
 
   before(function () {
     angular.module('app', [])
-      .component('topBar', require('../top-bar'))
+      .component('topBar', proxyquire('../top-bar', {
+        '../util/is-third-party-service': fakeIsThirdPartyService,
```

Comment:
I'm really not a huge fan of mocking simple, pure utility functions. I personally would have preferred just to create representative settings for the first and third party cases which could be as simple as:

```js
// First party
fakeSettings = {
  services: [{ authority: 'hypothes.is'} ],
  authDomain: 'hypothes.is',
};

// Third party
fakeSettings = {
  services: [{ authority: 'partner.org'} ],
  authDomain: 'hypothes.is',
};
```

However, this is subjective and we haven't hashed out a clear policy either way, so I'm OK to leave it as is.